requirements-docs.txt

```diff
@@ -1,8 +1,8 @@
-sphinx == 3.1.2
+sphinx
 sphinxcontrib-apidoc
 pbr
 setuptools >= 30.3.0
 
 numpy
-networkx ~= 2.0
+networkx
```

Comment:
I think defining a minimum networkx version would be better, because there are still quite few people that run older versions. But this is an optional comment

Reply:
@pckroon how about this one?

Reply:
I think latest is fine tbh, so let's leave it like this